src/seeds.dao.cpp

```diff
@@ -365,12 +365,14 @@ ACTION dao::erasepartpts (const uint64_t & active_proposals) {
 
   while (pitr != participants_t.end() && counter < batch_size) {
     if (pitr->count == active_proposals && pitr->nonneutral) {
-      send_inline_action(
-        permission_level(contracts::accounts, "addrep"_n),
-        contracts::accounts, 
-        "addrep"_n,
-        std::make_tuple(pitr->account, reward_points)
-      );
+      if (reward_points > 0) {
```

Comment:
this could be outside the look actually but under normal circumstances we expect the value to be > 0